distributed/ddp-tutorial-series/slurm/setup_pcluster_slurm.md

```diff
@@ -0,0 +1,51 @@
+# Setup AWS cluster with pcluster
+
+## 1. Sign in to an AWS instance
+
+## 2. Install pcluster
+```
+pip3 install awscli -U --user
+pip3 install "aws-parallelcluster" --upgrade --user
+```
+
+## 3. Create a cluster config file
+```
+pcluster configure --config config.yaml
+```
+See config.yaml.template for an example
```

Comment:
Does pcluster support command line arguments as well so we can parametrize the YAML?

Reply:
Not sure what you mean

Reply:
See this example https://github.com/pytorch/data/tree/main/benchmarks/cloud

Reply:
Hm, looks like only other arg it takes is `region`. Ref: https://docs.aws.amazon.com/parallelcluster/latest/ug/pcluster.configure.html